definitions/infra-squid_cachemgr/definition.yml

```diff
@@ -3,15 +3,19 @@ type: SQUID_CACHEMGR
 
 synthesis:
   rules:
-    - identifier: squid_server
-      name: squid_server
+    - identifier: squid_cachemgr
+      name: squid_cachemgr
       conditions:
         - attribute: eventType
           value: MetricRaw
         - attribute: metricName
           prefix: squid_
       tags:
-        squid_server:
+        squid_cachemgr:
```

Comment:
what is the point of this tag? if the entity already uses it as the name?

Reply:
You are right, I have removed it in the last fix.